CMakeLists.txt

```diff
@@ -29,7 +29,14 @@ include(CTest)
 
 cmake_minimum_required(VERSION 2.6)
 
-add_definitions("-D_CRT_SECURE_NO_DEPRECATE")
+option(BDWGC_INSTALL_CORD "Install cord" ON)
+option(BDWGC_SKIP_HEADERS "Skip installing header files" OFF)
+option(BDWGC_SKIP_TESTS "Skip building tests" OFF)
+
```

Comment:
Any comments on the names of these options?
I namespaced them with a 'BDWGC_' prefix, but these prefix is not present in other options, e.g.: `enable_gc_debug`.

Also, would it be a good idea to move all of the options to the top of the file?

Right now all of the options are declared just before being used, which makes them a bit hard to find.

Reply:
> Any comments on the names of these options?

It is better to remove BDWGC_ prefix.

> Any comments on the names of these options?

Sure. Could you please rebase your PR and do it? (It is OK to do in the same PR or open another one)